examples/utils/example-utils/src/modelLoader/modelContainerRuntimeFactory.ts

```diff
@@ -35,7 +35,7 @@ export abstract class ModelContainerRuntimeFactory<ModelType> implements IRuntim
 
 	/**
 	 * @param registryEntries - The data store registry for containers produced
-	 * @param runtimeOptions - The runtime options passed to the ContainerRuntime when instantiating it
+	 * @param runtimeOptions - The runtime options passed to the IContainerRuntime when instantiating it
```

Comment:
Nit
```suggestion
	 * @param runtimeOptions - The runtime options passed to the {@link @fluidframework/container-runtime#IContainerRuntime} when instantiating it
```

Reply:
Beyond the improved generated docs experience, leveraging links will help inform us when code references become stale.